docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+
+services:
+  # IPFS container definition
+  ipfs:
+    image: ipfs/go-ipfs:v0.7.0
+    env_file:
+      - .env-pinner
+    volumes:
+      - "ipfs-repo:/data/ipfs"
+    ports:
+      - 4001:4001
+      - 4001:4001/udp
+    expose:
+      - 5001
+    networks:
+      - rif-pinner-services
+
+  # Pinner container definition
+  rif-pinner-service:
+    image: rif-pinning-service:latest
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
```

Comment:
I think it would be better to move this to a envfile too.

Reply:
Well I liked to think about these settings as "hardcoded", eq. ones that you won't be changing in order to run it. While into the `.env-pinner` should go environment-specific configuration and other user's specific configuration.